### PR TITLE
Minor fix to index.js Params docs, Number instead of Array

### PR DESCRIPTION
**Pull request by chuanhao01**

### CODE REVIEW

**Comment by chuanhao01** [comment]
I think the bonus question given ans for the thrid (3rd) example is wrong.
Should be 3 instead of 2.

File: index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const { array } = require('yargs');
 
 /**
  *
- * @param {Array} input an Array of number
+ * @param {Number} input an Input Number
```
[the remark this answers is not in the document]
Comment (jeremiah-ang) [reply index.js:5]: The param is correct. let's revert this.

File: index.js [hunk 2]
```diff
@@ -105,7 +105,7 @@ function question5(input) {}
  * examples:
  * input = [], depth = 0
  * input = [[], 1], depth = 1
- * input = [[], [[], [1, []]]], depth = 2
+ * input = [[], [[], [1, []]]], depth = 3
```
[the remark this answers is not in the document]
Comment (jeremiah-ang) [reply index.js:108]: Good catch

File: index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (chuanhao01) [reply index.js:5]:
```
test('Test 1', () => {
		const input = 4;
		const expected = [4, 12, 44, 156];
		const result = question(input);
		expect(result).toEqual(expected);
	});
```

Caused I saw in the test case that input was just a Number, not an Array of numbers.
Unless I'm missing something

[the remark this answers is not in the document]
Comment (chuanhao01) [reply index.js:5]: Maybe change to something like this?
```javascript
 * @param {Number} The input Number
```

[the remark this answers is not in the document]
Comment (jeremiah-ang) [reply index.js:5]: Oh my bad, I thought it was for the bonus question, good catch.

**Comment by jeremiah-ang** [comment]
LGTM

**Pull request merged**